frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/obc-test.spec.ts

```diff
@@ -0,0 +1,140 @@
+import { expect } from 'chai';
+import {
+  ACCESS_KEY,
+  BOUND,
+  MASK,
+  MINUTE,
+  NS,
+  OBC_NAME,
+  OBC_STORAGE_CLASS,
+  SECOND,
+  SECRET_KEY,
+} from '../utils/consts';
+import { testDeployment, deployment } from '../mocks/deploymentData';
+import { CreateOBCHandler } from '../views/obcPage';
+import { testName } from '../../../integration-tests-cypress/support/index';
+
+describe('Test Object Bucket Claim resource', () => {
+  let obcHandler;
+  let obcUrl;
+
+  before(() => {
+    cy.url().then((url) => {
+      if (url.match(/oauth/)) {
+        cy.login();
+      }
+    });
+    cy.visit('/');
+    cy.install();
+    obcHandler = new CreateOBCHandler(OBC_NAME, testName, OBC_STORAGE_CLASS, '');
+    obcHandler.createBucketClaim();
+    cy.exec(`echo '${JSON.stringify(deployment)}' | kubectl create -n ${testName} -f -`);
+    cy.url().then((url) => {
+      obcUrl = url;
+    });
+  });
+
+  beforeEach(() => {
+    cy.wait(10 * SECOND);
+  });
+
+  it('Test if Object Bucket Claim is created sucessfully', () => {
+    cy.visit(obcUrl);
+    cy.get('.co-resource-item__resource-name').should('be.visible');
+    cy.byLegacyTestID('resource-title')
+      .invoke('text')
+      .should('eq', OBC_NAME);
+    cy.get('.co-resource-item__resource-status', { timeout: 1 * MINUTE })
+      .should('be.visible')
+      .invoke('text')
+      .should('eq', BOUND);
+  });
+
+  it('Test if owner and creation date are shown correctly', () => {
+    obcHandler.getOwnerAndDate();
+  });
+
+  it('Test if secret data is available', () => {
```

Comment:
Does this need to be a different `it` block?

Reply:
Merged this in `Test if secret data is masked` block.